src/cryptonote_core/uptime_proof.cpp

```diff
@@ -46,9 +46,11 @@ Proof::Proof(
         public_ip{sn_public_ip},
         storage_https_port{sn_storage_https_port},
         storage_omq_port{sn_storage_omq_port},
-        qnet_port{quorumnet_port} {
+        qnet_port{quorumnet_port},
+        git_version(OXEN_VERSION_TAG) {
```

Comment:
This feels a bit misnamed, because this version tag is often not a git version at all: e.g. for deb snapshot builds this will be something like "v11.0.6-1~ubuntu2004-dev", and similarly for release builds.  The git tag (and 
GenVersion.cmake machinery) only really applies when doing a local git repo build.

Perhaps rename to `"version_tag"`?